lib/vrl/tests/tests/expressions/arithmetic/merge/invalid.vrl

```diff
@@ -0,0 +1,29 @@
+# result:
```

Comment:
Pro-tip: if you look at the other `invalid.vrl` examples, you can do the following:

```ruby
# result: ~
#
# can only merge objects
# can only merge objects
# can only merge objects
```

(note the `~`)

Basically, this allows you to validate that an error with that title occurred, but prevents you from having to update the UI tests every time a minor detail in the diagnostic is updated.

We still have `tests/diagnostics` where you'd add _one_ full example of this error to make sure any UI changes are still caught there.

Reply:
Nice.